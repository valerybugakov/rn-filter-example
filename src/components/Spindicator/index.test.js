import React from 'react'
import renderer from 'react-test-renderer'

import Component from './index'

it('renders without crashing', () => {
  const value = {
    interpolate: () => {},
  }

  const tree = renderer.create(<Component value={value} />).toJSON()

  expect(tree).toMatchSnapshot()
})
