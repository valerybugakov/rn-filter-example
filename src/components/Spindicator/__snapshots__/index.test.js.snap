// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<View
  collapsable={undefined}
  style={
    Object {
      "transform": Array [
        Object {
          "rotate": undefined,
        },
      ],
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    ðŸ™ƒ
  </Text>
</View>
`;
